tools/loader/Loader.cpp

```diff
@@ -371,38 +410,278 @@ static void getModelInputs(std::vector<std::string> &inputNames,
+      node = F->createTranspose("NCHW2NHWC", node, NCHW2NHWC);
+      continue;
+    }
+
+    // Image NHWC to NCHW conversion.
+    if (funcName == "NHWC2NCHW") {
+      node = F->createTranspose("NHWC2NCHW", node, NHWC2NCHW);
+      continue;
+    }
+
+    // Image channel order inversion.
+    if ((funcName == "RGB2BGR") || (funcName == "BGR2RGB")) {
+      int numDim = (int)node->getType(0)->dims().size();
+      int lastDim = (int)node->getType(0)->dims().back();
+      CHECK(lastDim == 3) << strFormat(
+          "Graph string processor: for '%s' the tensor last dimension "
+          "must be 3 (encountered value is %d)!",
+          funcName.c_str(), lastDim);
+      node = F->createFlip(funcName, node, numDim - 1);
+      continue;
+    }
+
+    // Image RGB to YUV conversion.
+    if (funcName == "RGB2YUV") {
+      int lastDim = (int)node->getType(0)->dims().back();
+      CHECK(lastDim == 3) << strFormat(
+          "Graph string processor: for '%s' the tensor last dimension "
+          "must be 3 (encountered value is %d)!",
+          funcName.c_str(), lastDim);
+      // TODO: Implement conversion node.
+      LOG(FATAL) << "Not supported yet ...";
+      continue;
+    }
+
+    // Image YUV to RGB conversion.
+    if (funcName == "YUV2RGB") {
+      int lastDim = (int)node->getType(0)->dims().back();
+      CHECK(lastDim == 3) << strFormat(
+          "Graph string processor: for '%s' the tensor last dimension "
+          "must be 3 (encountered value is %d)!",
+          funcName.c_str(), lastDim);
+      // TODO: Implement conversion node.
+      LOG(FATAL) << "Not supported yet ...";
+      continue;
+    }
+
+    // TopK operator.
+    if (funcName == "TOPK") {
+      int k = func.getArgInt(0);
+      node = F->createTopK(funcName, node, k);
+      continue;
+    }
+
+    // Operator not supported.
+    LOG(FATAL) << strFormat(
+        "Graph string processor: operator '%s' is not supported!",
+        funcName.c_str());
+  }
+  return node;
+}
+
 std::unique_ptr<ProtobufLoader> Loader::loadModel() {
 
-  // Get model input names and types.
+  // Get loader function.
+  Function *F = getFunction();
+
+  // Get model input names, types and options.
   std::vector<std::string> inputNames;
   std::vector<Type> inputTypes;
-  getModelInputs(inputNames, inputTypes);
+  std::vector<std::string> inputOpts;
+  getModelInputs(inputNames, inputTypes, inputOpts);
+
+  // Model input name and type references.
   std::vector<const char *> inputNameRefs;
   std::vector<TypeRef> inputTypeRefs;
   for (size_t idx = 0, e = inputNames.size(); idx < e; idx++) {
     inputNameRefs.push_back(inputNames[idx].c_str());
     inputTypeRefs.push_back(&inputTypes[idx]);
   }
 
+  // Get model output names and options.
+  std::vector<std::string> outputNames;
+  std::vector<std::string> outputOpts;
+  getModelOutputs(outputNames, outputOpts);
+
+  // Create the input subgraphs.
+  std::vector<Node *> inputGraphs(inputOpts.size(), nullptr);
```

Comment:
`createSimpleGraphFromString()` now return one NodeValue: the intent is to have a graph with one input and one output otherwise attaching a multiple input or output graph to one placeholder is ambiguous.